cmd/influx_inspect/verify/seriesfile/command.go

```diff
@@ -0,0 +1,119 @@
+// Package seriesfile verifies integrity of series files.
+package seriesfile
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+
+	"github.com/influxdata/influxdb/logger"
+	"go.uber.org/zap/zapcore"
+)
+
+// Command represents the program execution for "influx_inspect verify-seriesfile".
+type Command struct {
+	Stdout io.Writer
+	Stderr io.Writer
+
+	dir        string
+	db         string
+	seriesFile string
+	verbose    bool
+	parallel   int
+}
+
+// NewCommand returns a new instance of Command.
+func NewCommand() *Command {
+	return &Command{
+		Stdout: os.Stdout,
+		Stderr: os.Stderr,
+	}
+}
+
+// Run executes the command.
+func (cmd *Command) Run(args ...string) error {
+	fs := flag.NewFlagSet("verify-seriesfile", flag.ExitOnError)
+	fs.StringVar(&cmd.dir, "dir", filepath.Join(os.Getenv("HOME"), ".influxdb", "data"),
+		"Data directory.")
+	fs.StringVar(&cmd.db, "db", "",
+		"Only use this database inside of the data directory.")
+	fs.StringVar(&cmd.seriesFile, "series-file", "",
+		"Path to a series file. This overrides -db and -dir.")
+	fs.BoolVar(&cmd.verbose, "v", false,
+		"Verbose output.")
+	fs.IntVar(&cmd.parallel, "p", runtime.GOMAXPROCS(0),
+		"How many parallel workers to run.")
+
+	fs.SetOutput(cmd.Stdout)
+	fs.Usage = cmd.printUsage
+
+	if err := fs.Parse(args); err != nil {
+		return err
+	}
+
+	config := logger.NewConfig()
+	if cmd.verbose {
+		config.Level = zapcore.DebugLevel
```

Comment:
The reason I went with DEBUG is because INFO was on by default. I needed some level below INFO that was off, that could be turned on with verbose. Thoughts?

Reply:
Ah I see. A couple of ways this gets done:

  - two loggers. One logs to stderr one to stdout. `-v` turns on the stdout one.
  - Only use the logger for the verbose stuff (at `INFO` level). So that means `-v` initialises a logger and its absence means we use the `NOP` one. Error messages get emitted via `fmt.FprintX`.

Or we just stick with what you have done, which on reflection seems completely fine. @benbjohnson what do you think?

Reply:
@e-dard What about having `-v` set level to `INFO` and no `-v` sets the level to `WARN`? Or just leave it. That works too.

Reply:
All sounds good to me